packages/webapp/src/components/ChartSection/Markup.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import trimValues from '../../helpers/trimValues';
+import Icon from '@material-ui/icons/ArrowForward';
+import SectionHeading from '../SectionHeading';
+import CssBaseline from '@material-ui/core/CssBaseline';
+
+import {
+  DetailsWrapper,
+  LinkWrapper,
+  ButtonStyle,
+  TextExploreButton,
+  SpanStyled,
+  DetailsContainer,
+  Department,
+  Amount,
+  ChartWrapper,
+  ChartContainer,
+  FooterWrapper,
+  FooterContainer,
+  FooterDetails
+ } from './styled';
+
+ const callChart = (selected, Chart, onSelectedChange) => (
+   <ChartWrapper>
+    <ChartContainer>
+      {Chart({selected, onSelectedChange})}
```

Comment:
You were actually right previously, this should only be `onSelectedChange` only. Just pass the later without an object.